apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/component/config/PortalConfig.java

```diff
@@ -273,4 +275,12 @@ public String getAdminServiceAccessTokens() {
   public boolean supportSearchByItem() {
     return getBooleanProperty("searchByItem.switch", true);
   }
+  
+  public List<String> listOfCodeFragment() {
+    String[] value = getArrayProperty("auth.user-password-checker.list-of-code-fragment", null);
```

Comment:
Add the prefix to the key?
```suggestion
    String[] value = getArrayProperty("apollo.portal.auth.user-password-checker.list-of-code-fragment", null);
```
Alternative
* auth.user-password-checker.list-of-code-fragment
* apollo.portal.auth.user-password-checker.list-of-code-fragment
* apollo.portal.auth.user-password-fragment
* apollo.portal.auth.user-password-checker-fragment
* apollo.portal.auth.user-password-checker-code-fragment
* apollo.portal.auth.user-password-not-allow-list
* ...

An better key let user more understandable this feature.

Reply:
Done.